mkosi/__init__.py

```diff
@@ -1872,6 +1872,11 @@ def build_uki(
         "--ro-bind", stub, stub,
     ]
 
+    if context.config.devicetree:
+        dtb = context.root / f"usr/lib/linux-image-{kver}" / context.config.devicetree
```

Comment:
This seems like a completely arbitrary documentation to me for putting device trees. Is there any kind of spec mandating that this is where device trees should be installed? If not, we need such a spec first. See for example https://github.com/uapi-group/specifications/pull/91 where a standard location for UKI addons is proposed. We'd need such a proposal for device trees as well.

Until then, I'd prefer if you used a ukify config file in a package manager tree to accomplish this.